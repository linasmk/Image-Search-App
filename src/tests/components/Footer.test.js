/* ========= App Dependencies ============= */
import React from "react";
import { shallow } from "enzyme";
/* ========= Components ============= */
import Footer from "@comp/Footer";
/* ========= Code ============= */
let wrapper;

beforeEach(() => {
  wrapper = shallow(<Footer />);
});

it("renders Footer component correctly", () => {
  expect(wrapper).toMatchSnapshot();
});

it("has HTML footer element", () => {
  expect(wrapper.find("footer").length).toEqual(1);
});

it("has HTML h4 element", () => {
  expect(wrapper.find("h4").length).toEqual(1);
});
