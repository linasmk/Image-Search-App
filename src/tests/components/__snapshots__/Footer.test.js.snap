// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component correctly 1`] = `
<footer
  className="footer"
>
  <h4
    className="footer__copyright"
  >
    Â© 
    2021
     Linas Mackonis
  </h4>
</footer>
`;
